### PR TITLE
Sra 114

### PR DESCRIPTION
**Pull request by telugucherla**
Hi Jim,

Can you please review.

Ravi

### CODE REVIEW

File: src/app/views/spots/sra-container.component.html [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+        <p>Account<br><strong>Mc Donald</strong></p>
+    </div>
+    <div class="col3">
+        <p>Markets<br><strong>Houstan TX, Colarodor Springs</strong></p>
+    </div>
+    <div class="col4">
+        <p>Last Parsed Data from<br><strong>July 16, 2018 12.00 pm CST</strong></p>
+    </div>
+    <div class="clearfix"></div>
+</div>
```
Comment (designbyjim) [inline src/app/views/spots/sra-container.component.html]: @telugucherla 
I think we'll want to remove this hard-coded markup

File: src/app/views/spots/sra-container.component.scss [hunk 2]
```diff
@@ -19,7 +19,50 @@ ag-grid-angular {
...
+}
+.clearfix{
+  clear:both;
+}
   button + button {
     margin-left: 15px;
   }
-}
\ No newline at end of file
+}
```
Comment (designbyjim) [inline src/app/views/spots/sra-container.component.scss]: @telugucherla 
looks like an extra trailing bracket?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (designbyjim) [inline .gitignore]: @telugucherla 
Thanks for noticing this – I guess I had a global `.gitignore` in effect.

File: src/app/views/spots/sra-container.component.html (hunk 1 above)
Comment (designbyjim) [reply src/app/views/spots/sra-container.component.html]: ignore previous comment here – I thought this was part of dev code for styling the table, I see now it's the strata ID section! (We'll just have to loop back and make it dynamic a la SRA-113)

**Comment by michael-ctp** [comment]
Verified ok

**Pull request merged**